### PR TITLE
Add a devtools button to roomsettings 

### PR DESCRIPTION
**Pull request by Half-Shot**
Fixes vector-im/riot-web#5151

### CODE REVIEW

File: src/components/views/rooms/RoomSettings.js [hunk 1]
```diff
@@ -942,6 +947,10 @@ module.exports = React.createClass({
             </AccessibleButton>;
         }
 
+        const devtoolsButton = <AccessibleButton className="mx_RoomSettings_devtoolsButton" onClick={this._openDevtools}>
+            { _t("Open Devtools") }
+        </AccessibleButton>;
```
Comment (turt2live) [inline src/components/views/rooms/RoomSettings.js]: Can we get it behind a boolean settings flag so non-powerusers aren't confused?

**Comment by turt2live** [comment]
also updating the US strings shouldn't be required, and I'd recommend using `npm run i18n` to generate the en strings to avoid it being re-sorted later.

File: src/components/views/rooms/RoomSettings.js (hunk 1 above)
Comment (Half-Shot) [reply src/components/views/rooms/RoomSettings.js]: It is stuck right under advanced? I don't know if a settings flag is going to do much to stop confusion.

**Comment by Half-Shot** [comment]
Noted, will do.

File: src/components/views/rooms/RoomSettings.js (hunk 1 above)
Comment (turt2live) [reply src/components/views/rooms/RoomSettings.js]: For reference (for you, or others, or no one):

* Add a setting to `Settings.js` - autoplay GIFs is a good one to copy for this ( https://github.com/matrix-org/matrix-react-sdk/blob/167746f41d891d053643e061a583489f2826755a/src/settings/Settings.js#L138-L142 )
* Add the setting to `UserSettings.js` - https://github.com/matrix-org/matrix-react-sdk/blob/167746f41d891d053643e061a583489f2826755a/src/components/structures/UserSettings.js#L63-L85
* Use the setting in an if statement like `if (SettingsStore.getValue("yourSettingName"))`

**Comment by t3chguy** [comment]
LGTM other than UX query of whether or not it should be in Advanced or hidden behind a settings toggle
@lampholder

File: src/components/views/rooms/RoomSettings.js (hunk 1 above)
Comment (turt2live) [reply src/components/views/rooms/RoomSettings.js]: gah, github only now showed me your comment, sorry.

Even though it is under advanced, people will probably still click it and become confused.

Comment (Half-Shot) [reply src/components/views/rooms/RoomSettings.js]: Yes, I wrote the autoplay GIFs feature so I know how settings works :p

**Comment by Half-Shot** [comment]
(Agreed OOB to include a "Show developer tools" button)

**Pull request merged**